src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -152,6 +152,28 @@ public function compileDropIfExists(Blueprint $blueprint, Fluent $command)
     }
 
     /**
+     * Compile a drop all tables command.
+     *
+     * @param  string  $tables
+     * @return string
+     */
+    public function compileDropAllTables($tables)
+    {
+        return 'drop table '.implode(',', $tables).' cascade';
```

Comment:
I dont think so. There are other sections in the same class that use a table name without escaping - such as `compileColumnListing($table)`. 

So if it should be changed, it should be changed everywhere - and thus be a different PR for that separate issue